### PR TITLE
update remaining renewable capacities (IRENA)

### PR DESCRIPTION
**Pull request by KabelWlan**
It was not super necessary but I updated the remaining renewable capacities with the new IRENA report and I also updated the sources.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
You should use the original source and value for the Sweden solar data as it's the official government source. (It also reports a higher installed value for solar)

**Comment by KabelWlan** [comment]
I changed it back. Thanks for your feedback.

File: config/zones.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
     "capacity": {
       "biomass": 1,
       "hydro": 0,
-      "solar": 1883,
-      "wind": 1
+      "solar": 2705,
+      "wind": 0
```
Comment (madsnedergaard) [inline config/zones.json:37]: Oh no, what happened to the [one wind turbine](https://u.ae/en/information-and-services/environment-and-energy/water-and-energy/types-of-energy-sources/wind-energy)? 😢

**Review by madsnedergaard — APPROVED** [review]
Nice, thanks for doing this! 💪

**Pull request merged**